Directory.Build.props

```diff
@@ -1,24 +1,25 @@
 ﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <WarningLevel>5</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-  </PropertyGroup>
+    <PropertyGroup>
+        <WarningLevel>5</WarningLevel>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    </PropertyGroup>
 
-  <!-- NuGet Metadata -->
-  <PropertyGroup>
-    <Description>A declarative command line and XML configuration parser for F# applications.</Description>
-    <Authors>Eirik Tsarpalis</Authors>
-    <Copyright>2019</Copyright>
-    <PackageTags>F#, argument, commandline, parser</PackageTags>
-    <!-- NOTE while ReproducibleBuilds/SourceLink derives the same value, other stuff in the FAKE fsx require this explicitly-->
-    <RepositoryUrl>https://github.com/fsprojects/Argu/</RepositoryUrl>
-    <PackageProjectUrl>https://fsprojects.github.io/Argu/</PackageProjectUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-  </PropertyGroup>
-  
-  <!-- FsDocs -->
-  <PropertyGroup>
-    <FsDocsLicenseLink>https://github.com/fsprojects/Argu/blob/master/LICENSE.md</FsDocsLicenseLink>
-    <FsDocsReleaseNotesLink>https://github.com/fsprojects/Argu/blob/master/RELEASE_NOTES.md</FsDocsReleaseNotesLink>
-  </PropertyGroup>
+    <!-- NuGet Metadata -->
```

Comment:
autoformatted all of these, which means indent of 4
say the word if want me to add a .editorconfig and/or go back to 2?

Reply:
Yes, please add an `.editorconfig` for this. Rider tends to shuffle things around, would be nice if we have this configured.

Reply:
Indent of 2 seems most common, even if 4 is more legible esp given C#/F# tend to be 4.
Should I make it 2 ?

Reply:
Made it 2